test/models/user_test.rb

```diff
@@ -26,6 +26,12 @@ def setup
     refute User.new(email: test_email).valid?
   end
 
+  test 'email must be saved downcased' do
+    new_email = 'NEW_EMAIL@exaMple.cOm'
+    assert create(:user, email: new_email.downcase)
+    refute User.new(email: new_email).valid?
+  end
+
```

Comment:
Perhaps add a test that specifically confirms that the email saved to the database is downcased?  Ping me if you'd like any ideas on that.